test/src/main/java/org/apache/accumulo/test/compaction/CompactionExecutorIT.java

```diff
@@ -180,6 +212,18 @@ public static void teardown() {
 
   @AfterEach
   public void cleanup() {
+    var iter = compactionGroups.iterator();
+    while (iter.hasNext()) {
+      var group = iter.next();
+      getCluster().getConfig().getClusterServerConfiguration().addCompactorResourceGroup(group, 0);
+      try {
+        getCluster().getClusterControl().stopCompactorGroup(group);
+      } catch (Exception e) {
+        // Exception is encountered but ignored.
```

Comment:
Why ignore this?  Was wondering if it would be useful to log it.

Reply:
Added logging statement for the exception.